opencti-platform/opencti-front/src/private/components/common/stix_core_relationships/StixCoreRelationshipOverview.jsx

```diff
@@ -241,13 +241,31 @@ class StixCoreRelationshipContainer extends Component {
           classes={{ container: classes.gridContainer }}
         >
           <Grid item xs={6}>
-            <Typography variant="h4" gutterBottom={true}>
+            <Typography variant="h4" gutterBottom={false} sx={{ float: 'left' }}>
               {t('Relationship')}{stixCoreRelationship.draftVersion && (<DraftChip/>)}
             </Typography>
+            {!stixCoreRelationship.is_inferred && (
+            <Security needs={[KNOWLEDGE_KNUPDATE]}>
+              <IconButton
+                color="primary"
+                onClick={this.handleOpenEdition.bind(this)}
+                size="large"
+                sx={{ margin: '-15px 0px 0px -2px' }}
```

Comment:
This is ugly, but the all grid use highly unstable css with float and all. ^^ I tried to put a simple `<Stack> `component and it was breaking all layout 💥